pacsman/integration_test.py

```diff
@@ -127,12 +128,16 @@ def test_remote_patient_search(remote_client):
 @pytest.mark.remote
 def test_remote_series_for_study(remote_client):
     # this series is for patient PAT014
-    series_datasets = remote_client.series_for_study('1.2.826.0.1.3680043.11.119')
+    series_datasets = remote_client.series_for_study('1.2.826.0.1.3680043.11.118')
     assert len(series_datasets) > 1
 
 
 @pytest.mark.remote
 def test_remote_fetch_fail(remote_client):
+    # Skip failure check for dummy client (which never fails)
```

Comment:
This test is the only one that fails (as it expects to fail). The filesystem client just ignores the URL and port you give it and acts like it does for the local tests. I could add a separate `remote_clients_list`. It would make it a little more confusing for adding more clients later on but it would avoid the special case here.

Reply:
Ahh, I see--if the other tests pass then I think it makes sense to leave as is.